### PR TITLE
PackageChecksumRules

### PR DESCRIPTION
**Pull request by hlaueriksson**
Implements #4

### CODE REVIEW

**Comment by neilenns** [comment]
I took a quick peek, it looks reasonable. I pulled the branch locally and ran it against my DiscordTimestamp repo and it passed. Here's what it looked like in the debugger:

image

I suggest adding a note to the readme or something to go along with this rule that says a checksums.txt file can be generated in GitHub workflows using the `wangzuo/action-release-checksums@v1` action (https://github.com/wangzuo/action-release-checksums) otherwise nobody will know about this.

**Comment by hlaueriksson** [comment]
Yes, the README should be updated.

**Comment by hlaueriksson** [comment]
@neilenns I updated the README. Does it make sense?

File: README.md [hunk 1]
```diff
@@ -114,6 +117,40 @@ Logs are written to a file, `ptrun-lint.log`, in the same directory as the tool
...
+- Run the `releasenotes.ps1` script from [Community.PowerToys.Run.Plugin.Templates](https://github.com/hlaueriksson/Community.PowerToys.Run.Plugin.Templates)
+
+Example:
+
+| Filename | SHA256 Hash
+| --- | ---
+| `Install-0.1.0-arm64.zip` | `CB03EF645F24248F9618AF18E82D7BA7127F209AB1BE77701FC6183FDC952037`
+| `Install-0.1.0-x64.zip` | `E0DB7B8A98D0891B97AAF85A68D340EDDCDF09389537BA64401CF8D786746EC5`
+
+Create a checksums file:
```
Comment (neilenns) [inline README.md]:
```suggestion
Create a checksums file and attach it to the release as an asset:
```

File: README.md [hunk 2]
```diff
@@ -114,6 +117,40 @@ Logs are written to a file, `ptrun-lint.log`, in the same directory as the tool
...
+- Filename: `checksums.txt`
+- Method: `SHA256`
+
+Example:
+
+```txt
+CB03EF645F24248F9618AF18E82D7BA7127F209AB1BE77701FC6183FDC952037 Install-0.1.0-arm64.zip
+E0DB7B8A98D0891B97AAF85A68D340EDDCDF09389537BA64401CF8D786746EC5 Install-0.1.0-x64.zip
+```
+
```
Comment (neilenns) [inline README.md:153]:
```suggestion

A `checksum.txt` can be generated automatically in a GitHub workflow using the [wangzuo/action-release-checksums action](https://github.com/wangzuo/action-release-checksums).
```

**Review by neilenns — COMMENTED** [review]
Added a couple of readme suggestions.

**Pull request merged**